Marlin/Configuration.h

```diff
@@ -206,6 +206,21 @@
   #define PARKING_EXTRUDER_SECURITY_RAISE 5           // Z-raise before parking
 #endif
 
+/**
+* Separated Tool Heads with extruders which uses a Docking Station
+* Lock and Unlock via Servo
+*/
+//#define SWITCHING_TOOLHEAD
+#if ENABLED(SWITCHING_TOOLHEAD)
+#define SWITCHING_TOOLHEAD_SERVO_NR 2
+#define SWITCHING_TOOLHEAD_SERVO_ANGLES { 0, 180 }		// 1. Lock Angle, 2. Unlock Angle
+#define SWITCHING_TOOLHEAD_Y_POS	235					// Y position of the tool head dock
```

Comment:
Tabs are forbidden. Please set your editor to always use 2 real spaces for indentation.

Reply:
Do you have a recommendation for a good editor? I actually using xcode. It handled the spaces very bad...